src/MagnumPlugins/DdsImporter/Test/DdsImporterTest.cpp

```diff
@@ -89,15 +94,6 @@ void DdsImporterTest::unknownFormat() {
     CORRADE_COMPARE(out.str(), "Trade::DdsImporter::openData(): unknown format\n");
 }
 
-void DdsImporterTest::unknownCompression() {
-    std::ostringstream out;
-    Error redirectError{&out};
-
-    DdsImporter importer;
-    CORRADE_VERIFY(!importer.openFile(Utility::Directory::join(DDSIMPORTER_TEST_DIR, "unknown_compression.dds")));
-    CORRADE_COMPARE(out.str(), "Trade::DdsImporter::openData(): unknown compression DX10\n");
-}
```

Comment:
This would make the "unknown compression" error branch uncovered. I think we should still check this to make sure we handle corrupted files well. Is it possible to craft such a file that fits the criteria?